sql/structured.proto

```diff
@@ -73,6 +73,13 @@ message IndexDescriptor {
   // parallels the column_names list.
   repeated uint32 column_ids = 5 [(gogoproto.customname) = "ColumnIDs",
       (gogoproto.casttype) = "ColumnID"];
+  // An ordered list of additional column ids to use for the index. For
```

Comment:
Doesn't the rest of the comment provide adequate clarification? If not, I need clarification on what sort of clarification you need.